src/Endatix.Api/Endpoints/Assistant/DefineForm.cs

```diff
@@ -0,0 +1,51 @@
+using FastEndpoints;
+using MediatR;
+using Microsoft.AspNetCore.Http.HttpResults;
+using Endatix.Api.Infrastructure;
+using Endatix.Infrastructure.Identity.Authorization;
+using Endatix.Core.UseCases.Assistant.DefineForm;
+
+namespace Endatix.Api.Endpoints.Assistant;
+
+/// <summary>
+/// Endpoint for defining a form using AI assistance.
+/// </summary>
+public class DefineForm(IMediator _mediator) : Endpoint<DefineFormRequest, Results<Ok<DefineFormResponse>, BadRequest>>
```

Comment:
Primary constructors inject mutable parameters, so we should not use underscore prefix like we do this with private readonly fields

I think, this is now added to the text-editor config. Please change

Reply:
Yes, this is how they work. I don't think we have agreed on the underscore usage as a prefix. Personally, I don't like it for any fields but we have it in almost all of the endpoints currently. I just follow what we have.
Let's decide our naming conventions and make the change everywhere, not having it inconsistent.